dataland-frontend/src/components/forms/CreateEuTaxonomyForFinancials.vue

```diff
@@ -201,81 +201,78 @@
-                    :name="financialServiceOption.value"
-                    type="group"
-                  >
-                    <div
-                      v-for="kpiType of euTaxonomyKPIsModel[financialServiceOption.value]"
-                      :key="kpiType"
-                      :data-test="kpiType"
-                      class="uploadFormSection"
-                    >
-                      <div class="col-9 formFields">
-                        <FormKit :name="kpiType" type="group">
-                          <div class="form-field">
-                            <DataPointForm
-                              :name="kpiType ?? ''"
-                              :kpiInfoMappings="euTaxonomyKpiInfoMappings"
-                              :kpiNameMappings="euTaxonomyKpiNameMappings"
-                              :reportsName="namesOfAllCompanyReportsForTheDataset"
-                            />
+                    <FormKit :name="financialServiceOption.value" type="group">
+                      <div v-if="financialServiceOption.value !== 'assetManagementKpis'" class="uploadFormSection">
+                        <div
+                          v-for="kpiType of euTaxonomyKPIsModel[financialServiceOption.value]"
+                          :key="kpiType"
+                          :data-test="kpiType"
+                          class="uploadFormSection"
+                        >
+                          <div class="col-9 formFields">
+                            <FormKit :name="kpiType" type="group">
+                              <div class="form-field">
+                                <DataPointForm
+                                  :name="kpiType ?? ''"
+                                  :kpiInfoMappings="euTaxonomyKpiInfoMappings"
+                                  :kpiNameMappings="euTaxonomyKpiNameMappings"
+                                  :reportsName="namesOfAllCompanyReportsForTheDataset"
+                                />
+                              </div>
+                            </FormKit>
                           </div>
-                        </FormKit>
+                        </div>
                       </div>
-                    </div>
-                  </FormKit>
-
-                  <FormKit name="eligibilityKpis" type="group">
-                    <FormKit
-                      :name="euTaxonomyKPIsModel?.kpisFieldNameToFinancialServiceType[financialServiceOption.value]"
-                      type="group"
-                    >
-                      <div
-                        v-for="kpiTypeEligibility of euTaxonomyKPIsModel.eligibilityKpis"
-                        :key="kpiTypeEligibility"
-                        :data-test="kpiTypeEligibility"
-                        class="uploadFormSection"
+                      <FormKit
+                        :name="euTaxonomyKPIsModel?.kpisFieldNameToFinancialServiceType[financialServiceOption.value]"
```

Comment:
We would save us some complexity if the name of this formkit group would just be eligibilityKpis

Reply:
Oh, it doesn't. But it would make things clearer IMO.

Reply:
But back then `elibilityKpis` existed multiple times as one object. Here it would be different subobjects